pkg/clusteragent/custommetrics/provider.go

```diff
@@ -71,6 +74,16 @@ func (p *datadogProvider) ListAllExternalMetrics() []provider.ExternalMetricInfo
 	var externalMetricsInfoList []provider.ExternalMetricInfo
 	var externalMetricsList []externalMetric
 
+	copyAge := metav1.Now().Unix() - p.timestamp
+	if copyAge < p.maxAge { //&& p.externalMetrics != nil{
+		log.Tracef("Local copy is recent enough, not querying the GlobalStore. Remaining %d/%d seconds before next sync", copyAge, p.maxAge)
```

Comment:
the `Remaining %d/%d seconds` thing is strange, why not printing `maxAge - copyAge`?

Reply:
yeah, I did that for another log you're right